src/index.ts

```diff
@@ -79,12 +96,17 @@ export class Storage<BodyType extends Document, MetaType extends Document> {
     this.accessControl = accessControl || new DefaultAccessControl<BodyType, MetaType>()
     this.changelog = changelog || new DefaultChangelog(staticChangelog)
     this.searchIndex = index || new NeDbSearchIndex<BodyType, MetaType>({ logger: this.logger })
-    this.currentState = state || new LocalState<BodyType, MetaType>({ size: 50000, searchIndex: this.searchIndex })
+    this.currentState = state || new LocalState<BodyType, MetaType>({
+      size: config.cacheSize || 5000000,
```

Comment:
I kind of feel like this should be `currentState.cacheSize`, but maybe that's just me.

Reply:
mkay

Reply:
actually... if we used Redis as a state adapter, .size wouldn't make much sense, would it?